albatross/core/model_adapter.h

```diff
@@ -112,7 +114,12 @@ class AdaptedRegressionModel
 
   void unchecked_set_param(const std::string &name,
                            const double value) override {
-    sub_model_.set_param(name, value);
+
+    if (map_contains(this->params_, name)) {
+      this->params_[name] = value;
```

Comment:
nit: What's the difference between `params_[name] = value` and `.set_param(name, value)`?

Reply:
`.set_param(name, value)` checks to make sure the parameter exists, then calls `unchecked_set_param`: https://github.com/swift-nav/albatross/blob/master/albatross/core/parameter_handling_mixin.h#L74-L77.  so if I called it from here we'd get a circular reference.